dsl/src/main/scala/org/http4s/dsl/impl/ResponseGenerator.scala

```diff
@@ -76,10 +76,10 @@ trait LocationResponseGenerator[F[_], G[_]] extends Any with EntityResponseGener
   @deprecated("Use `apply(Location(location))` instead", "0.18.0-M2")
   def apply(location: Uri)(implicit F: Applicative[F]): F[Response[G]] =
     F.pure(
-      Response[G](status = status, headers = Headers(`Content-Length`.zero, Location(location))))
+      Response[G](status = status, headers = Headers(`Content-Length`.zero :: Location(location) :: Nil)))
```

Comment:
Do you find this preferrable to `Headers.of`?

Reply:
I originally was just going to deprecate, but realized it's used readily so transitioned.